### ISSUE TITLE
what is the official way to run js unit test cases with nuclide。

### ISSUE THREAD

**Issue by Kerence** [facebook/nuclide#1084]
I'm longing for a manual to run js unit test cases with nuclide.
the following page show the button of 'toggle test runner',but there's no further info for how to use test runner the official way.
https://nuclide.io/docs/features/toolbar/#buttons
### Actual Behavior
I have figured out a way to run js unit tests with test runner in an abrupt way.
I installed nuclide-test-runner-jest and runs unit test in __test__ folder, whereas the test runner shows both passed tests and failed tests in both gray color which slows error finding.
### Expected Behavior
I suppose there is an official and better way to run unit tests with nuclide

### Versions
* Atom:1.15.0
* Nuclide:0.214.0
* Client OS:osx 10.12.3
* Server OS (optional):

**Comment by petegleeson** [comment]
Yeah I am having the same difficulty at the moment. I would love to see some documentation on how to run es6 tests (whether it is mocha, jest, etc) with debugger support.

**Comment by kyledecot** [comment]
I'm also curious how to do this.
